_includes/methods/markdown/design-hypothesis.md

```diff
@@ -1,14 +1,22 @@
 ## How to do it
 
 1. As a team, identify and make explicit the problem you’re trying to solve. What goals or needs aren’t being met? What measurable criteria would indicate progress toward those goals?
-1. As a team, write out the hypothesis for the work you want to do to address the problem(s) you’re trying to solve. You may want to write broad hypotheses at the outset of a project and more specific hypotheses each sprint.
-    <br/>Here’s a common way to structure your hypothesis:
-
-    - *We believe that doing/building/creating [this] for [this user] will result in [this outcome].*
-    - *We’ll know we’re right when we see [this metric/signal].*
 
+1. As a team, write out the hypothesis for the work you want to do to address the problem(s) you’re trying to solve. You may want to write broad hypotheses at the outset of a project and more specific hypotheses each sprint.
+    <p>
+      Here’s a common way to structure your hypothesis:
+    </p>
+    <p>
+      <em>We believe that doing/building/creating [this] for [this user] will result in [this outcome].</em>
+    </p>
+    <p>
+      <em>We’ll know we’re right when we see [this metric/signal].</em>
+    </p>
 1. Once you've formulated your hypothesis, consider the following harm prompt to help the team think about and guard against potential unintended consequences of your work.
-    - *But, this could be harmful for [this user] if [this outcome happens].*
+
+    <p>
```

Comment:
I don't know if p tags are technically allowed in ordered lists....but also most browsers will play along with it?  I don't know.  I guess the other option is to use a `span` that's style as a display-block or something?

Reply:
I'm not considering this a blocking problem, for the record

Reply:
Thanks—I was trying to keep parity with the rendered html of the old site, which produces p tags.